### PR TITLE
Add Redecorating's fork for backlight

### PR DESCRIPTION
**Pull request by AdityaGarg8**

### CODE REVIEW

**Comment by Redecorating** [comment]
it doesn't work on mba9,1, also probably mention that this is to get the keyboard backlight working

**Comment by AdityaGarg8** [comment]
I've still added MBA 9,1 cause ur fork is made for that too. Also, I didn't add backlight cause this guide is for someone who is setting up linux for the first time and not for making something fix the backlight

**Comment by AdityaGarg8** [comment]
@Redecorating , please review.

File: docs/guides/dkms.md [hunk 1]
```diff
@@ -43,7 +43,14 @@ sudo rm -r /var/lib/dkms/apple-ibridge
 
 2. Installing the Touchbar and Ambient Light sensor modules
 
-    - Run `sudo git clone https://github.com/t2linux/apple-ib-drv /usr/src/apple-ibridge-0.1`
+    - If you are on a MacBook Pro (16 inch, 2019), MacBook Pro (13 inch, 2020), or MacBook Air (Retina, 13-inch, 2020), and want keyboard backlight to work, run :
```
Comment (Redecorating) [inline docs/guides/dkms.md:46]: It doesn't work on MBA9,1. I've updated my fork to remove it from the ID list.

**Review by Redecorating — APPROVED** [review]

**Pull request merged**